build.gradle

```
@@ -696,13 +696,14 @@ if (HUDSON_JOB_NAME == "not_hudson") {
     relOpt = "-${buildTimestamp}"
 } else {
     if (IS_MILESTONE_FCS) {
-        if (jfxExperimentalReleaseSuffix != "") {
-            fail("An experimental release branch should not be used for FCS build")
+        if (jfxExperimentalFeatureName != "") {
+            fail("An experimental feature branch should not be used for FCS build")
         }
         relSuffix = ""
     } else {
-        relSuffix = jfxExperimentalReleaseSuffix != "" ?
-                        jfxExperimentalReleaseSuffix : jfxReleaseSuffix;
+        relSuffix = jfxExperimentalFeatureName != "" ?
+                        "-${jfxExperimentalFeatureName}" : jfxReleaseSuffix;
+        relOpt = "-${buildTimestamp}"
```

This will unconditionally add a timestamp to _all_ EA builds of JavaFX, not just experimental builds. Changing the version string for EA builds seems out of scope for this PR.

```suggestion
        if (jfxExperimentalFeatureName != "") {
            relSuffix = "-${jfxExperimentalFeatureName}"
            relOpt = "-${buildTimestamp}"
        } else {
            relSuffix = jfxReleaseSuffix
        }
```

---

Thanks @kevinrushforth , Included the change.